src/components/button/button.spec.js

```diff
@@ -177,67 +175,11 @@ describe('Button', () => {
     );
   });
 
-  describe.each(classicColorVariants)(
-    'when the color variant is set to "%s"',
-    (variant) => {
-      const wrapper = TestRenderer.create(
-        <StyledButton
-          size='large'
-          theme={ classicTheme }
-          variant={ variant }
-        >Foo
-        </StyledButton>
-      );
-
-      it('matches the snapshot with the default props', () => {
-        expect(wrapper).toMatchSnapshot();
-      });
-    }
-  );
-
-  describe('when the classic theme is applied', () => {
-    describe.each(classicColorVariants)(
-      'setting the color variant to "%s"',
-      (variant) => {
-        const wrapper = TestRenderer.create(
-          <StyledButton
-            iconType='services'
-            theme={ classicTheme }
-            variant={ variant }
-          >Foo
-          </StyledButton>
-        );
-
-        it('matches the snapshot when default props are passed', () => {
-          expect(wrapper).toMatchSnapshot();
-        });
-      }
-    );
-
-    it('matches the expected style for a "blue" Button with default props', () => {
-      const wrapper = TestRenderer.create(<StyledButton theme={ classicTheme }>foo</StyledButton>);
-      assertStyleMatch({
-        background: 'transparent',
-        border: '1px solid #255bc7',
-        color: '#255bc7'
-      }, wrapper.toJSON());
-    });
-
-    it('matches the expected style default "disabled" Button', () => {
-      const wrapper = TestRenderer.create(<StyledButton disabled theme={ classicTheme }>foo</StyledButton>);
-      assertStyleMatch({
-        background: '#e6ebed'
-      }, wrapper.toJSON());
-    });
-  });
-
-  describe('when not in classic theme', () => {
-    it('matches the applies the expected style to the icon', () => {
-      const wrapper = TestRenderer.create(<StyledButton iconType='plus' />);
-      assertStyleMatch({
-        height: '16px'
-      }, wrapper.toJSON(), { modifier: `${StyledIcon}` });
-    });
+  it('matches the applies the expected style to the icon', () => {
```

Comment:
```suggestion
  it('applies the expected style to the icon', () => {
```

Reply:
or matches if that fits better

Reply:
This is just a refactor of an existing test. I'll look to tidy the test suite in a future PR.